### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by aruizgar21**

### CODE REVIEW

**Comment by ocorcho** [comment]
Next time, please do it using git instead of the GitHub interface

File: Assignment2/aruizgar21-AntonioRuiz.ttl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://ex.org/#> .
```
Comment (ocorcho) [inline Assignment2/aruizgar21-AntonioRuiz.ttl:2]: there should be a prefix name

File: Assignment2/aruizgar21-AntonioRuiz.ttl [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://ex.org/#> .
+:Class01 rdfs:includes:Computer101;
```
Comment (ocorcho) [inline Assignment2/aruizgar21-AntonioRuiz.ttl:3]: Space should be added between includes and :Computer101

Comment (ocorcho) [reply Assignment2/aruizgar21-AntonioRuiz.ttl:3]: And rdfs:includes is wrong.

File: Assignment2/aruizgar21-AntonioRuiz.ttl [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://ex.org/#> .
+:Class01 rdfs:includes:Computer101;
+rdfs:includes:Sensor029.
+:Sensor029 rdfs:hasMeasurement:Measurement8401.
+:Measurement8401 rdfs:hasTemperature "29";
```
Comment (ocorcho) [inline Assignment2/aruizgar21-AntonioRuiz.ttl:6]: You should add a datatype

File: Assignment2/aruizgar21-AntonioRuiz.ttl [hunk 4]
```diff
@@ -0,0 +1,10 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://ex.org/#> .
+:Class01 rdfs:includes:Computer101;
+rdfs:includes:Sensor029.
+:Sensor029 rdfs:hasMeasurement:Measurement8401.
+:Measurement8401 rdfs:hasTemperature "29";
+rdfs:atTime "2010-06-12T12:00:12".
```
Comment (ocorcho) [inline Assignment2/aruizgar21-AntonioRuiz.ttl:7]: You should add a datatype

File: Assignment2/aruizgar21-AntonioRuiz.ttl [hunk 5]
```diff
@@ -0,0 +1,10 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://ex.org/#> .
+:Class01 rdfs:includes:Computer101;
+rdfs:includes:Sensor029.
+:Sensor029 rdfs:hasMeasurement:Measurement8401.
+:Measurement8401 rdfs:hasTemperature "29";
+rdfs:atTime "2010-06-12T12:00:12".
+:Computer101 rdfs:hasOwner:User10A.
```
Comment (ocorcho) [inline Assignment2/aruizgar21-AntonioRuiz.ttl:8]: Same problem with the space

File: Assignment2/aruizgar21-AntonioRuiz.ttl [hunk 6]
```diff
@@ -0,0 +1,10 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://ex.org/#> .
+:Class01 rdfs:includes:Computer101;
+rdfs:includes:Sensor029.
+:Sensor029 rdfs:hasMeasurement:Measurement8401.
+:Measurement8401 rdfs:hasTemperature "29";
+rdfs:atTime "2010-06-12T12:00:12".
+:Computer101 rdfs:hasOwner:User10A.
+:User10A rdfs:hasName "Pedro" ;
```
Comment (ocorcho) [inline Assignment2/aruizgar21-AntonioRuiz.ttl:9]: Will it parse?

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Review by aruizgar21 — COMMENTED** [review]
Changed

**Pull request merged**